vendor/engines/projects/app/models/projects/project.rb

```diff
@@ -0,0 +1,17 @@
+module Projects
+
+  class Project < ActiveRecord::Base
+
+    include ActiveModel::ForbiddenAttributesProtection
+
+    belongs_to :facility, foreign_key: :facility_id
+    attr_accessible :description, :name
```

Comment:
With `ActiveModel::ForbiddenAttributesProtection`, you don't need `attr_accessible` anymore.